src/includes/load-service.h

```diff
@@ -150,18 +150,100 @@ inline string_iterator skipws(string_iterator i, string_iterator end) noexcept
     return i;
 }
 
-// Convert a signal name to the corresponding signal number
-inline int signal_name_to_number(std::string &signame) noexcept
+// Utilities to map strings/signals to int
+// The signal list is created statically with only those signals,
+// which are defined for the current architecture/system
+std::pair<const char *, int> signal_to_int_map[] = {
```

Comment:
I wrote this draft new implementation and saw some weird stuff (mainly `ld` reported that `signal_to_int_map` duplicated multiple times). Seems like it's needed to be `inline`ed to work correctly but inline variables are C++17 specific thing. Needs more investigation.

@davmac314 Did you mean something like this?

Reply:
You've removed `constexpr`, that's why you're getting multiple definition errors.

Reply:
(other than that, yes, that's how the definition should look, but the signal list still needs to be trimmed down)